src/LanguageServer/Impl/Protocol/Classes.cs

```diff
@@ -585,13 +585,21 @@ public sealed class SignatureInformation {
         public string label;
         public MarkupContent documentation;
         public ParameterInformation[] parameters;
-
-        public string[] _returnTypes;
     }
 
     [Serializable]
     public sealed class ParameterInformation {
-        public string label;
+        /// <summary>
+        /// The label of this signature.
+        /// </summary>
+        /// <remarks>
+        /// LSP before 3.14: string label.
+        /// LSP 3.14.0+: (int, int) range.
+        /// Either a string or inclusive start and exclusive end offsets within its containing
+        /// [signature label] (#SignatureInformation.label). *Note*: A label of type string must be
+        /// a substring of its containing signature information's [label](#SignatureInformation.label).
+        /// </remarks>
+        public object label;
```

Comment:
So they have made a change that breaks backward compatibility?

Reply:
Not really. In earlier than 3.14 it is `string`. 3.14+ it is [number, number]. In the initialization info the client tells if it supports offsets rather than strings and server can then provide them. Strings are still accepted in any case.

Reply:
Do we have tests for both cases?

Reply:
No. We actually never really verified bolding until now. Should we?

Reply:
Let me try add one

Reply:
Added